resources/locales/de/translation.json

```diff
@@ -1 +1,8 @@
-{}
+{
+  "notifications": {
+    "checkNumberSeatExceeded": {
+      "title": "Anzahl Sitze überprüfen",
+      "body": "Die Anzahl Sitze der einzelnen Parteien ergeben mehr Sitze als die gesamte Sitzanzahl des Parlament."
```

Comment:
Small spelling error at the end. Also, the notification pop ups when I clear the input field, which seems a little confusing.